service/src/test/java/uk/gov/hmcts/reform/fpl/service/payment/PaymentServiceTest.java

```diff
@@ -147,6 +147,32 @@ void shouldMakeCorrectPaymentForC2WithNoticeWhenCustomerReferenceIsNullOrEmpty(
             verify(feeService).getFeesDataForC2(WITH_NOTICE);
         }
 
+        @ParameterizedTest
```

Comment:
You covered WITH_NOTICE / WITHOUT_NOTICE but the c110a payment is not covered here by tests. This needs to be addressed. 

Would it be worth actually parametrising the WITH/WITHOUT? Just a thought. Don't need to address this one.

Reply:
1. Regarding c110 payment, I think I need to have a word with you to understand more.
2. WITH and WITHOUT has now been addressed by parameterising which is a good suggestion and in long run it will be helpful.

Reply:
I see you added c110a tests below, seems to cover my comments.